Userland/Libraries/LibGfx/Font/WOFF/Font.cpp

```diff
@@ -120,12 +121,18 @@ ErrorOr<NonnullRefPtr<Font>> Font::try_load_from_externally_owned_memory(Readonl
         if (font_buffer_offset + orig_length > font_buffer.size())
             return Error::from_string_literal("Uncompressed WOFF table too big");
         if (comp_length < orig_length) {
-            auto decompressed = Compress::ZlibDecompressor::decompress_all(buffer.slice(offset, comp_length));
-            if (!decompressed.has_value())
-                return Error::from_string_literal("Could not decompress WOFF table");
-            if (orig_length != decompressed->size())
+            auto compressed_data_stream = TRY(try_make<FixedMemoryStream>(buffer.slice(offset, comp_length)));
+            auto decompressor_or_error = Compress::ZlibDecompressor::create(move(compressed_data_stream));
+            if (decompressor_or_error.is_error())
```

Comment:
Why aren't you using a `TRY()`?

Reply:
Why indeed. I'm sure I thought that I had a reason for doing that.